### PR TITLE
Added current epoch and next epoch pool stats to pools/id endpoint, a…

### PR DESCRIPTION
**Pull request by alexkroeger**
Added `PoolWithStats` info to the pools/`id` endpoint.

Added timestamps to the pool-epoch rewards array for graphing.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -109,9 +109,9 @@ export interface RawPoolEpochRewards {
     starting_block_timestamp: string;
     starting_block_number: string;
     starting_transaction_index: string;
-    ending_block_number?: string;
-    ending_timestamp?: string;
-    ending_transaction_hash?: string;
+    ending_block_number: string;
+    ending_timestamp: string;
+    ending_transaction_hash: string;
```
Comment (fragosti) [inline src/types.ts]: hmmm how? we use this type for other queries too, where these can be undefined / null.

**Comment by johnrjj** [comment]
I'm getting `QueryFailedError: bind message supplies 1 parameters, but prepared statement "" requires 0` on `GET /staking/pools/1` when running locally

**Comment by johnrjj** [comment]
Pushed a fix

**Comment by johnrjj** [comment]
`currentEpochStats.operatorShare` and `nextEpochStats.operatorShare` are null , is that expected?

File: src/types.ts (hunk 1 above)
Comment (alexkroeger) [reply src/types.ts]: rewards only accrue for a finished epoch (known end time)

**Comment by alexkroeger** [comment]
> `currentEpochStats.operatorShare` and `nextEpochStats.operatorShare` are null , is that expected?

I'm seeing values in the query. Can you give a pool ID, network that you're using?

**Pull request merged**